backend/GymBeam/Controllers/ActivityController.cs

```diff
@@ -8,6 +8,7 @@
 using Microsoft.AspNetCore.Mvc;
 using System.Net;
 using Domain.Exceptions;
+using GymBeam.Constants;
```

Comment:
this is needed in case of switch to Release(then we use Const to get Const.Roles)